src/ExtendFinSearch.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace FINDOLOGIC\ExtendFinSearch;
+
+use FINDOLOGIC\FinSearch\FinSearch;
+use Shopware\Core\Framework\Plugin;
+use Shopware\Core\Framework\Plugin\Context\InstallContext;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+
+class ExtendFinSearch extends Plugin
+{
+    public function build(ContainerBuilder $container): void
+    {
+        require_once $this->getBasePath() . '/vendor/autoload.php';
+        parent::build($container);
+
+        $activePlugins = $container->getParameter('kernel.active_plugins');
+        if (!isset($activePlugins[FinSearch::class])) {
```

Comment:
This case can and should never happen. Why do we need to take care of it here?

Reply:
It needs to be here because we do not want to load the `finsearch_services.xml` if the plugin is not installed and active. Otherwise it will throw an exception about the class being dependent on a non-existing service. 
That is why the service is not defined in the regular `services.xml` as it will get loaded by default upon installation.